pkg/omf/functions/util/omf.check.version.fish

```diff
@@ -0,0 +1,3 @@
+function omf.check.version -a min_version -a current_version
+  test (echo "$min_version"\n"$current_version" | sort -V | head -n1) = "$min_version"
```

Comment:
_my_ version comes up with the right answer when checking `omf.check.version 3.11.0 3.11` … but i guess that's an edge case that'll probably never be needed :P